Scheduler/v6/maintel_fbs_sit_block_t87.yaml

```diff
@@ -0,0 +1,69 @@
+s3instance: cp
```

Comment:
so, this might be problematic if you try to run this on TTS or BTS, maybe remove it?